client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -496,7 +509,8 @@ def read(node):
             except json.JSONDecodeError:
                 # not a json
                 pass
-        data[parameter.name()] = value
+        # Use parameter name without prefix if prefix exists.
+        data[parameter.name().replace(KEY_PREFIX, "", 1)] = value
```

Comment:
This should not be done here - much better to prepare the data as needed inside the Creator plug-ins. Again, this is `lib` so I'd expect the functions to be somewhat generic and re-usable.